### PR TITLE
[new release] current_examples, current, current_ansi, current_slack, current_git, current_web, current_github, current_docker and current_rpc (0.1)

### PR DESCRIPTION
**Pull request by talex5**
Example pipelines for OCurrent

- Project page: <a href="https://github.com/ocurrent/ocurrent">https://github.com/ocurrent/ocurrent</a>
- Documentation: <a href="https://ocurrent.github.io/ocurrent/">https://ocurrent.github.io/ocurrent/</a>

##### CHANGES:

Initial release.

### CODE REVIEW

File: packages/current/current.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "ocaml" {>= "4.08.0"}
+  "fmt"
+  "bos"
+  "ppx_deriving"
+  "lwt"
+  "cmdliner"
+  "sqlite3"
+  "duration"
+  "prometheus"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current/current.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_ansi/current_ansi.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "astring"
+  "fmt"
+  "tyxml"
+  "fpath" {with-test}
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_ansi/current_ansi.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_docker/current_docker.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "current" {= version}
+  "current_git" {= version}
+  "ocaml" {>= "4.08.0"}
+  "fmt"
+  "ppx_deriving"
+  "lwt"
+  "dockerfile"
+  "ppx_deriving_yojson" {>= "3.5.1"}
+  "yojson"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_docker/current_docker.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_examples/current_examples.0.1/opam [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  "cmdliner"
+  "duration"
+  "current" {= version}
+  "current_web" {= version}
+  "current_git" {= version}
+  "current_github" {= version}
+  "current_docker" {= version}
+  "current_rpc" {= version}
+  "capnp-rpc-unix" {>= "0.5"}
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_examples/current_examples.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_git/current_git.0.1/opam [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+depends: [
+  "current" {= version}
+  "ocaml" {>= "4.08.0"}
+  "fmt"
+  "ppx_deriving"
+  "lwt"
+  "irmin-watcher"
+  "ppx_deriving_yojson" {>= "3.5.1"}
+  "yojson"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_git/current_git.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_github/current_github.0.1/opam [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  "current_git" {= version}
+  "ocaml" {>= "4.08.0"}
+  "fmt"
+  "lwt"
+  "yojson"
+  "cohttp-lwt-unix"
+  "nocrypto"
+  "x509" {>= "0.7.0"}
+  "tls"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_github/current_github.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_rpc/current_rpc.0.1/opam [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+dev-repo: "git+https://github.com/ocurrent/ocurrent.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "capnp-rpc-lwt" {>= "0.4"}
+  "fpath"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_rpc/current_rpc.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_slack/current_slack.0.1/opam [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+]
+depends: [
+  "current" {= version}
+  "ocaml" {>= "4.08.0"}
+  "fmt"
+  "yojson"
+  "lwt"
+  "tls"
+  "cohttp-lwt-unix"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_slack/current_slack.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

File: packages/current_web/current_web.0.1/opam [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  "current_ansi" {= version}
+  "ocaml" {>= "4.08.0"}
+  "fmt"
+  "bos"
+  "lwt"
+  "cmdliner"
+  "prometheus-app"
+  "cohttp-lwt-unix" {>= "2.2.0"}
+  "tyxml"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/current_web/current_web.0.1/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Your `dune-project` file says this package requires at least dune 1.9

**Comment by talex5** [comment]
I've applied the suggested changes - thanks!

Summary of CI failures:
- Travis: both builds timed-out after 50 min
- Alpine and CentOS failed because capnp isn't available on those platforms.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**